packages/@uppy/companion/src/server/provider/dropbox/index.js

```diff
@@ -20,6 +20,27 @@ function httpHeaderSafeJson (v) {
     })
 }
 
+function adaptData (res, email, buildURL) {
+  const items = adapter.getItemSubList(res).map((item) => ({
+    isFolder: adapter.isFolder(item),
+    icon: adapter.getItemIcon(item),
+    name: adapter.getItemName(item),
+    mimeType: adapter.getMimeType(item),
+    id: adapter.getItemId(item),
+    thumbnail: buildURL(adapter.getItemThumbnailUrl(item), true),
+    requestPath: adapter.getItemRequestPath(item),
+    modifiedDate: adapter.getItemModifiedDate(item),
+    size: adapter.getItemSize(item),
+  }))
+  items.sort((a, b) => a.name.localeCompare(b.name, 'en-US', { numeric: true }))
```

Comment:
```suggestion
  })).sort((a, b) => a.name.localeCompare(b.name, 'en-US', { numeric: true }))
```

Reply:
i think it reads better on a separate line

Reply:
I don't, and also it's not immediately obvious that `.sort` would modify the original array if you're not familiar. I won't push it if that's your preference though.